index.html

```diff
@@ -461,16 +467,89 @@ <h5 class="head-text-1">LET'S CONNECT</h5>
                 class="collapsible about-collapsible-button head-text-2"
               >
                 Skills
-                <object
-                  data="assets/icons/chevron.svg"
-                ></object>
+                <object data="assets/icons/chevron.svg"></object>
               </button>
             </div>
             <hr class="divider-1" />
           </li>
-
         </ul>
       </div>
     </section>
+
+    <!--CONTACT AREA-->
+    <section id="ContactForm" class="contact-section">
+      <div class="contact-background">
+        <img
+          class="illus-svg vector10"
+          src="assets/illustrations/vector-7.svg"
+          alt="###"
+        />
+        <img
+          class="illus-svg vector11"
+          src="assets/illustrations/vector-6.svg"
+          alt="###"
+        />
+        <img
+          class="illus-svg ellipse2"
+          src="assets/illustrations/ellipse-3.svg"
+          alt="###"
+        />
+      </div>
+
+      <div class="contact-stack">
+        <div class="contact-banner">
+          <h1 class="contact-header banner-content">Contact me</h1>
+          <p class="body-text-2 color-white banner-content">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+        </div>
+        <div class="contact-form">
+          <form action="https://formspree.io/f/mwkyyazl" method="post">
+            <label for="name">
+              <input
+                type="text"
+                name="name"
+                id="name"
+                required
+                maxlength="30"
+                class="contact-input"
+                placeholder="Enter your names"
+              />
+            </label>
+            <label for="email">
+              <input
+                type="email"
+                name="email"
+                id="email"
+                class="contact-input"
+                placeholder="Enter your gmail"
+                required
+              />
+            </label>
+            <label for="text">
+              <textarea
+                name="text"
+                id="text"
+                required
+                maxlength="30"
+                class="contact-input"
+                cols="23"
+                rows="5"
+              ></textarea>
```

Comment:
- [ ] Please edit the max length to be `"500"` instead of `"30"`, as per [this](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#:~:text=Message%20field%20accepts%20up%20to%20500%20chars.) requirement.
- [ ] Please make sure that the text area field has a placeholder that matches the Figma template.